releasenotes/notes/fixconnectiontruncation-a7fa7b4e5351b17e.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
```

Comment:
Do we want to add a section about the bump of the limit from 160 to 512?

Reply:
I think that's more of an internal thing than public facing, but happy to add that if you think it's relevant